Monika After Story/game/script-windowreacts.rpy

```diff
@@ -562,3 +562,1500 @@ label mas_wrs_twitch:
+    addEvent(
+        Event(
+            persistent._mas_windowreacts_database,
+            eventlabel="mas_wrs_camera",
+            category=['camera'],
+            rules={
+                "notif-group": "Window Reactions",
+                "skip alert": None,
+                "keep_idle_exp": None,
+                },
+            show_in_idle=True
+        ),
+        code="WRS"
+    )
+
+label mas_wrs_camera:
+    $ wrs_success = mas_display_notif(
+        m_name,
+        [
+           "Say cheese! Ahaha!"
+        ],
+        'Window Reactions'
+    )
+
+
+    if not wrs_success:
+        $ mas_unlockFailWRS('mas_wrs_camera')
+    return       
+
+init 5 python:
+    addEvent(
+        Event(
+            persistent._mas_windowreacts_database,
+            eventlabel="mas_wrs_netscape",
+            category=['welcometonetscape!'],
+            rules={
+                "notif-group": "Window Reactions",
+                "skip alert": None,
+                "keep_idle_exp": None,
+                },
+            show_in_idle=True
+        ),
+        code="WRS"
+    )
+
+label mas_wrs_netscape:
+    $ wrs_success = mas_display_notif(
+        m_name,
+        [
+           "Hmmm, an old guide to the Internet? Don't you already know how this all works, though?\nEhehe, just wondering!"
+        ],
+        'Window Reactions'
+    )
+
+
+    if not wrs_success:
+        $ mas_unlockFailWRS('mas_wrs_netscape')
+    return
+
+init 5 python:
+    addEvent(
+        Event(
+            persistent._mas_windowreacts_database,
+            eventlabel="mas_wrs_unity",
+            category=['unity'],
+            rules={
+                "notif-group": "Window Reactions",
+                "skip alert": None,
+                "keep_idle_exp": None,
+                },
+            show_in_idle=True
+        ),
+        code="WRS"
+    )
+
+label mas_wrs_unity:
+    $ wrs_success = mas_display_notif(
+        m_name,
+        [
+           "Are you designing your own game? Very cool!"
+        ],
+        'Window Reactions'
+    )
+
+
+    if not wrs_success:
+        $ mas_unlockFailWRS('mas_wrs_unity')
+    return
+
+init 5 python:
+    addEvent(
+        Event(
+            persistent._mas_windowreacts_database,
+            eventlabel="mas_wrs_gms2",
+            category=['gamemakerstudio2'],
+            rules={
+                "notif-group": "Window Reactions",
+                "skip alert": None,
+                "keep_idle_exp": None,
+                },
+            show_in_idle=True
+        ),
+        code="WRS"
+    )
+
+label mas_wrs_gms2:
+    $ wrs_success = mas_display_notif(
+        m_name,
+        [
+           "Are you designing your own game? Very cool!"
+        ],
+        'Window Reactions'
+    )
+
+
+    if not wrs_success:
+        $ mas_unlockFailWRS('mas_wrs_gms2')
+    return
+
+init 5 python:
+    addEvent(
+        Event(
+            persistent._mas_windowreacts_database,
+            eventlabel="mas_wrs_sfm",
```

Comment:
this isn't necessarily the only case for making a video. Unreal Engine is also something which can be used to make them. Among other things